templates/server/src/main/resources/archetype-resources/.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-Drevision=0.0.1 -Dchangelist=-SNAPSHOT
```

Comment:
An archetype can take a version as argument. That is available via `${version}` variable.
It will now be ignored and the initial version is hardcoded to this `0.0.1-SNAPSHOT`.
As `${version}` can also be `1.0.0-SNAPSHOT` we can not easily do
```
-Drevision=${version} -Dchangelist=-SNAPSHOT
```

As this would then result in `1.0.0-SNAPSHOT-SNAPSHOT`.
However, as a nice to have, we could check if via velocity macros it is possible to strip out a potential `-SNAPSHOT` from `${version}`. If that is possible, we should use `${version}` to make it perfect.
However, for me also fine to approve and merge like this.

Reply:
The formatting is done..

version passing as an argument would be really nice..

Reply:
Thanks for formatting. To prevent blocking this PR, I will merge it now.
Can you please create a new issue for the nice-to-have improvement to put the version here and research velocity macro syntax to remove potential `-SNAPSHOT` suffix?

Reply:
Issue #346 created.

is it okay to directly place the version to app.version properties ... like below

#if($version)
	<app.version>${version}</app.version>
#else
	<app.version>${revision}${changelist}</app.version>
#end